### ISSUE TITLE
New rule idea: Warn when a disable is unnecessary

### ISSUE THREAD

**Issue by j-f1** [mysticatea/eslint-plugin-eslint-comments#2]
For example:

```js
var x = 2;
/* eslint-disable no-undef */ // Error: `no-undef` rule is not triggered in this section.
x = 3
/* eslint-enable no-undef */
```

This verifies that you actually **need** all of your `eslint-disable*` comments.

**Comment by mysticatea** [comment]
Thank you for this issue.

However I'm confused. Is it different from [no-unused-disable](https://github.com/mysticatea/eslint-plugin-eslint-comments/blob/master/docs/rules/no-unused-disable.md) rule?

**Comment by j-f1** [comment]
No, that rule covers it. Thanks for pointing that out!
